cmd/os_mgmt/port/zephyr/src/zephyr_os_mgmt.c

```diff
@@ -91,6 +91,6 @@ zephyr_os_mgmt_reset_cb(struct k_timer *timer)
 int
 os_mgmt_impl_reset(unsigned int delay_ms)
 {
-    k_timer_start(&zephyr_os_mgmt_reset_timer, K_MSEC(delay_ms), 0);
+    k_timer_start(&zephyr_os_mgmt_reset_timer, K_MSEC(delay_ms), K_MSEC(0));
```

Comment:
I believe the standard spelling when specifying no repeat interval is `K_NO_WAIT`.  The value may not be the same as as `K_MSEC(0)` as it was when integer values were used.

Reply:
Indeed, please use `K_NO_WAIT` just like the other users int he tree (there seems to be a couple of odd ones out, I will send a PR to correct the doc.

Reply:
ok, done.